_content/wg.md

```diff
@@ -11,40 +11,47 @@ name: wg
 
-EIC has the unique opportunity to start incorporating AI from the very beginning and to systematically leverage on it during all phases of the project. AI will be an integral part of the EIC software and we will take advantage of intelligent decisions in all aspects of data processing from detector readout and control to analysis. To work in this direction, there is an AI Working Group (AI) as part of the SWG. The AIWG will serve as an entry point to AI applications and will organize workshops, tutorials, and Kaggle-like challenges. Conveners:
-* Cristiano Fanelli - AI sector - MIT - <cfanelli@mit.edu>
-* Tanja Horn - AI sector - CUA - <hornt@cua.edu>
+#### AI Working Group
 
-The group maintains a mailing list: <eicug-software-ai@eicug.org> and a ["AI4EIC website"](https://eic.ai/){:target="_blank"}.
+EIC has the unique opportunity to start incorporating AI from the very beginning and to systematically leverage on it during all phases of the project. AI will be an integral part of the EIC software and we will take advantage of intelligent decisions in all aspects of data processing from detector readout and control to analysis. To work in this direction, there is an AI Working Group (AI) as part of the SWG. The AIWG will serve as an [entry point to AI applications](https://eic.ai){:target="_blank"} and will organize workshops, tutorials, and Kaggle-like challenges. 
 
+##### Communication
+The SWG communicates via its [mailing list](mailto:eicug-software@eicug.org){:target="_blank"} and organizes regular [online and in-person meetings](https://indico.bnl.gov/category/301/){:target="_blank"} as well as tutorials that enable broad and active participation from within the EICUG as a whole. For questions about the Software Working Group, please contact the [conveners](mailto:eicug-software-conveners@eicug.org){:target="_blank"}: 
+* Andrea Bressan (University of Trieste and INFN)
+* Markus Diefenthaler (Jefferson Lab)
+* Cristiano Fanelli (Massachusetts Institute of Technology)
+* Tanja Horn (Catholic University of America)
+* Torre Wenaus (BNL)
 
 <span id="computing-coordination"><br/></span>
 
 ---
 
 ### Computing&nbsp;Coordination
 
-Conveners:
+The Computing Coordination Group coordinates resources among EIC computing efforts. Members are the liaisons for EIC computing resources at BNL and Jefferson Lab and representatives from the EICUG SWG and the forming EIC collaboration:
 * Jerome Lauret - BNL - <jeromel@bnl.gov>
 * Graham Heyes - JLab - <heyes@jlab.org>
+* More members will be added when the updated charge and membership for the Computing Coordination Group will be announced. 
```

Comment:
We should avoid these kind of 'promissory' statements since they change how viewers perceive the website. Essentially we make a promise but have no way of tracking it. A website should ideally be written to be 'timeless' and eternally valid. I think we could replace this bullet with
```suggestion
* Additional community representatives.
```
That's correct right now, it is correct when people are decided on but before we update, and it does not turn the website invalid when we do not update this.